Metro-Records/la-metro-councilmatic
# Refactor board members listing and search pages

## Overview
This continues the template upgrade to bootstrap 5, this time for the board members listing and search pages. This also makes some small bug fixes and changes. Some highlights are:

#### Search
- The 'remove' button that renders next to the 'order by' sorting options on the search page wasn't actually removing the sort. This has been fixed. A condition has also been added so that if the order is by 'relevance', then the remove doesn't render because that is the default order anyway.
- The panels that were used to show and hide facets have been upgraded to accordions. The style was changed along with this, so they don't look identical. IMO I don't mind the way they look and animate, but if we want it to be closer to the original then I can make some adjustments.
- The tags that show up with search results can get very long and don't wrap, causing the page to have extra space to the right if they're present. My solution here is to give those tags a max-width at smaller screen sizes with an ellipsis representing the overflow.

#### Board members listing
- The map on the board members page said "Tap ward to view details" on mobile sizes, implying that it would do the same as the hover on desktop. Tapping actually brought you to the member's detail page, so the language has been changed on mobile sizes to reflect this.
- The 'show more' buttons for the Recent Activity and Board of Directors Meetings sections have been moved to the top of those areas. This is to prevent users having to scroll to the bottom of a potentially long list of items to collapse the list.

This one seems to be a bit chunkier than the last, apologies! I'll try to submit these in smaller sections. Bring on any questions you have.

Connects #969 

### Notes

I left a couple todo's in there for now, that I do intend on going back to.

## Testing Instructions

- Check the index and about pages against their live versions on desktop and mobile window sizes
- Confirm that
  - they look presentable/similar to the original
  - the functionality across those two pages still works (maps, clear filter buttons, faceting etc)
